tests/test_secure_model.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+from tf_encrypted.private_model import secure_model
+
+
+class TestSecureModel(unittest.TestCase):
+
+    def test_secure_model(self):
+        tf.random.set_random_seed(42)
```

Comment:
seed is probably not necessary but i figured we would occasionally have precision errors without it.